### PR TITLE
Make use of init function and extract config package

### PR DESCRIPTION
**Pull request by kohpai**

### CODE REVIEW

File: config/database/init.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package database
+
+import (
+	"os"
+
```
Comment (oat9002) [inline config/database/init.go:5]: Please remove the space

File: config/init.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package config
+
+import (
+	"log"
+	"os"
+
```
Comment (oat9002) [inline config/init.go:6]: Please remove the space

File: main.go [hunk 3]
```diff
@@ -1,40 +1,15 @@
 package main
 
 import (
-	"fmt"
 	"log"
-	"os"
```
Comment (oat9002) [inline main.go:5]: remove space

**Review by oat9002 — CHANGES_REQUESTED** [review]
:2nd_place_medal:

File: config/database/init.go (hunk 1 above)
Comment (kohpai) [reply config/database/init.go:5]: You mean the new line? It's there just to separate between system's libs (Go built-in libraries) and external libs (e.g., from github).

**Comment by kohpai** [comment]
@oat9002 I should've got the gold medal!!!

**Review by oat9002 — APPROVED** [review]

**Comment by oat9002** [comment]
@kohpai :1st_place_medal:

**Pull request merged**